### PR TITLE
Fixes #34046 - UX Review - Spacing

### PR DESCRIPTION
**Pull request by Andrewgdewar**
#### What are the changes introduced in this pull request?

- Adjusted spacing/placement on all pages across the new CV UI, as well as a few changes on the new Hosts UI.
 
#### Considerations taken when implementing this change?

- "Consistency" - Maria 

#### What are the testing steps for this pull request?

- Navigate through all new  CV/Hosts pages, using [this](https://marvelapp.com/prototype/8813483/screen/80289005)  design template as an outline for spacing.

- Note any pages that look off or don't follow a similar pattern.

NOTE: Accompanying Foreman PR here for host details: https://github.com/theforeman/foreman/pull/8996

### CODE REVIEW

File: webpack/scenes/ContentViews/Copy/CopyContentViewForm.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ const CopyContentViewForm = ({ cvId, setModalOpen }) => {
         <Button
           aria-label="copy_content_view"
           variant="primary"
-          isDisabled={name.length <= 2 || saving}
+          isDisabled={!name?.length || saving}
```
Comment (Andrewgdewar) [inline webpack/scenes/ContentViews/Copy/CopyContentViewForm.js:64]: fixed

File: webpack/scenes/ContentViews/Create/CreateContentViewForm.js [hunk 2]
```diff
@@ -64,7 +64,7 @@ const CreateContentViewForm = ({ setModalOpen }) => {
     return (<Redirect to={`/content_views/${id}`} />);
   }
 
-  const submitDisabled = name.length <= 2 || label.length <= 2 || saving;
+  const submitDisabled = !name?.length || !label?.length || saving;
```
Comment (Andrewgdewar) [inline webpack/scenes/ContentViews/Create/CreateContentViewForm.js:67]: fixed

**Review by sjha4 — APPROVED** [review]
Tested and spacing changes look good on the UI. Ack 👍🏽

**Review by jeremylenz — APPROVED** [review]
Clicked & looked around, all looks fine to me!  ACK :+1:

**Pull request merged**